homework-04/routes/authentication.js

```diff
@@ -47,12 +51,14 @@ passport.use(new LocalStrategy({
   passwordField: 'password',
   session: false
 }, (username, password, done) => {
-  const user = usersDb.find(user => user.email === username)
-  if (user && user.password === password) {
-    done(null, user)
-  } else {
-    done(null, false, 'Invalid email or password')
-  }
+  User.findOne({where: {email: username, password: password}})
```

Comment:
in javascript we can use `{email: username, password}`

Reply:
fixed